discord/utils.py

```diff
@@ -1067,6 +1069,14 @@ def set_target(
                 except AttributeError:
                     pass
 
+
+def _generate_session_id() -> str:
+    return ''.join(
+        random.choice(string.ascii_uppercase + string.ascii_lowercase + string.digits)
+        for _ in range(16)
+    )
```

Comment:
Also didn't know that join() could be run without [] inside. As in, I would code this as
```python
return ''.join([
        random.choice(string.ascii_uppercase + string.ascii_lowercase + string.digits)
        for _ in range(16)
    ])
```
As I didn't know your implementation was possible, very cool!